NBA/server/routes/stats.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter,HTTPException
+from services.nba_api import fetch_stats
+
+router = APIRouter()
+
+@router.get('/',status_code=200)
+async def get_stats(last_name: str, first_name: str):
+    try:
+        stats = await fetch_stats(last_name,first_name)
```

Comment:
Should not call "services.nba_api" directly, should pass through "server.py" like you have done with "get_team"